gfauto/gfauto/amber_converter.py

```diff
@@ -214,7 +214,7 @@ def amberscript_uniform_buffer_bind(uniform_json: str, prefix: str) -> str:
     result = ""
     uniforms = json.loads(uniform_json)
     for name, entry in uniforms.items():
-        if name == "$compute":
+        if name[0] == "$":
```

Comment:
I prefer `name.startswith("$")`. Also, please update the doc comment above (since you did it already for the function below).